README.md

```diff
@@ -45,105 +43,103 @@ Start the application in interactive Elixir mode
 #### **Starting the miner**
 To start the miner use the following command in the command prompt:
 
-`Aecore.Miner.Worker.resume()`
+`Miner.resume()`
 
 This will continuously mine new blocks until terminated or suspended.
 To suspend/stop the miner from mining:
 
-`Aecore.Miner.Worker.suspend() `
+`Miner.suspend() `
 
 #### **Building custom child Transaction**
 To build a custom transaction you need to follow few simple steps:
-- Make your own `transaction module`;
-- Create your `custom transaction structure`;
-- Override the `Transaction Behaviour` callbacks;
-- Write all your specific functions and checks inside your new `Transaction module`;
+- Make your own `transaction module`
+- Create your `custom transaction structure`
+- Override the `Transaction Behaviour` callbacks
+- Write all your specific functions and checks inside your new `Transaction module`
 
 All custom transactions are childs to the `DataTx` Transaction that wraps them inside.
 The DataTx strucure hold:
-- The name of your `transaction type` that should be you `Transaction Module name`;
-- The `payload` that will hold your `custom transaction structure`;
-- The `sender`;
-- The `nonce`;
-- The `fee`;
+- The name of your `transaction type` that should be you `Transaction Module name`
```

Comment:
I know this was like this even before the PR but the sentence doesn't make it clear, if this is the atom of the module then maybe we should say something along those lines

Reply:
@gspasov you have a proposal for this?

Reply:
Maybe: __The name of your new `tramsacton type` should be the name of the `Transaction Module name`__ ?